{"status": "broken", "name": "Adding a list property", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "SetAcceptance"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486146087, "historyId": "68203935f3a8f213a6b750b79d8a87b9", "start": 1578486145862, "steps": [{"status": "passed", "start": 1578486145864, "stop": 1578486146077, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486146078, "stop": 1578486146080, "name": "And having executed", "attachments": [{"source": "4c056694-ebe0-46e4-8d5e-1c9aebde1ed0-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486146081, "stop": 1578486146083, "name": "When executing query", "attachments": [{"source": "bd6f57b8-c7d0-45f5-947c-a97049f9f792-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "de00fd70-9a91-4c6a-a54a-c7263d94c414-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486146085, "start": 1578486146084, "statusDetails": {"message": "ResponseError: RedisGraph does not currently support list comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486146086, "stop": 1578486146086, "name": "And the side effects should be", "attachments": [{"source": "a9c51281-4af2-4584-afae-b48be51c615a-attachment.csv", "type": "text/csv", "name": ".table"}]}], "statusDetails": {"message": "ResponseError: RedisGraph does not currently support list comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "0869e7f5-b6b6-4336-9dbb-7c2e13be261d"}